packages/yarnpkg-core/sources/Configuration.ts

```diff
@@ -335,7 +335,7 @@ export const coreDefinitions: {[coreSettingName: string]: SettingsDefinition} =
   enableTelemetry: {
     description: `If true, telemetry will be periodically sent, following the rules in https://yarnpkg.com/advanced/telemetry`,
     type: SettingsType.BOOLEAN,
-    default: !isCI,
+    default: true,
```

Comment:
Note that it isn't actually `true` - we simply moved the `isCI` check in the place where the telemetry manager is instantiated, so that it's disabled on CI even if explicitly enabled.